src/fuel_watcher.rs

```diff
@@ -1,163 +1,261 @@
-use crate::alerts::{AlertLevel, WatchtowerAlerts};
-use crate::ethereum_actions::WatchtowerEthereumActions;
+use crate::alerter::{AlertLevel, AlertParams, send_alert};
+use crate::ethereum_actions::{ActionParams, send_action};
 use crate::WatchtowerConfig;
+use crate::config::FuelClientWatcher;
+use crate::fuel_watcher::fuel_utils::get_value;
 
 use anyhow::Result;
-use fuel_chain::FuelChain;
-use fungible_token_contract::FungibleTokenContract;
+use fuel_chain::FuelChainTrait;
+
+use std::sync::Arc;
 use std::thread;
 use std::time::Duration;
 use tokio::task::JoinHandle;
+use tokio::sync::mpsc::UnboundedSender;
 
 pub mod fuel_chain;
-pub mod fungible_token_contract;
+pub mod fuel_utils;
+pub mod extended_provider;
 
-pub static POLL_DURATION: Duration = Duration::from_millis(4000);
+pub static POLL_DURATION: Duration = Duration::from_millis(10000);
```

Comment:
Any reason this was changed?

Reply:
Testing around, what do you think about making these polls configurable?